docker/nginx-default.conf

```diff
@@ -24,6 +24,13 @@ server {
 
     charset utf-8;
 
+    # In testing, Bytespider was very very spammy even against the testing
```

Comment:
Is there an easy way to keep such changes out of git, perhaps in an nginx dot directory or some such? It feels specific to packages.zeek.org (as opposed to, say, local testing/futzing with the container).

It reminds of of the chinese crawlers we fought a few back (though we handled those via iptables), so there's probably always going to be a bit of churn in these if we actually bother to look for them.